benchmarks/main.go

```diff
@@ -72,27 +68,29 @@ func testBenchmarkLakeFS() error {
 		"storage_namespace": ns,
 		"name":              repoName,
 	}).Debug("Create repository for test")
-	repo, err := client.Repositories.CreateRepository(repositories.NewCreateRepositoryParamsWithContext(ctx).
-		WithRepository(&models.RepositoryCreation{
-			DefaultBranch:    "master",
-			Name:             swag.String(repoName),
-			StorageNamespace: swag.String(ns),
-		}), nil)
+	createRepoResp, err := client.CreateRepositoryWithResponse(ctx, &api.CreateRepositoryParams{}, api.CreateRepositoryJSONRequestBody{
+		DefaultBranch:    api.StringPtr("master"),
+		Name:             repoName,
+		StorageNamespace: ns,
+	})
 	if err != nil {
 		return fmt.Errorf("failed to create repository, storage '%s': %w", ns, err)
 	}
+	if err := responseAsError("create repository", createRepoResp); err != nil {
+		return err
+	}
 
-	_, err = client.Branches.CreateBranch(&branches.CreateBranchParams{
-		Branch: &models.BranchCreation{
-			Source: swag.String("master"),
-			Name:   swag.String(branchName),
-		},
-		Repository: repo.Payload.ID,
-		Context:    ctx,
-	}, nil)
+	repo := createRepoResp.JSON201
```

Comment:
will have to update the client code template and support OK and Err which make it easier to use.

Reply:
Used WithResponses because I wanted the parsed responses.
Think we need to improve the client template next and make it as easy as possible to access the default OK/success response.